### PR TITLE
clickfunnels dc template

### PR DESCRIPTION
**Pull request by Thekkedath**
DC template for clickfunnels

### CODE REVIEW

File: goentri.com.clickfunnels.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      {
+        "type": "TXT",
+        "host": "%domain%",
+        "pointsTo": "clickfunnels-domain-verification=%verificationValue%",
+        "ttl": 3600
+      },
+      {
+        "type": "CNAME",
+        "host": "@",
+        "pointsTo": "%clusterValue",
```
Comment (pawel-kow) [inline goentri.com.clickfunnels.json:24]: It's `data` not `pointsTo` for TXT  record.

File: goentri.com.clickfunnels.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "ttl": 3600
+      },
+      {
+        "type": "TXT",
+        "host": "%websiteId%._domainkey",
+        "pointsTo": "%keyValue%",
```
Comment (pawel-kow) [inline goentri.com.clickfunnels.json:30]: It's `data` not `pointsTo` for TXT  record.

**Review by pawel-kow — CHANGES_REQUESTED** [review]

**Review by pawel-kow — APPROVED** [review]
LGTM!

**Comment by Thekkedath** [comment]
there was a missing % and variable description, so have updated that in the new PR

**Pull request merged**